src/utils/funksjonerForAmplitudeLogging.ts

```diff
@@ -26,7 +36,59 @@ export const loggSidevisning = (pathname: string, innlogget: Innlogget) => {
     });
 };
 
-export const loggBedriftValgtOgTilganger = (
+const finnAntallAnsattebøtte = (antall: number) => {
+    switch (true) {
+        case antall === 0:
```

Comment:
Vurdere kortere navn? `'0'`, `'1-4'`, `'5-19'`, `'20-49'`, osv. Og bare `'?'` eller `undefined` for siste.

Reply:
![Gif found on CodeReviewGifs.com hosted on Giphy.com](https://media.giphy.com/media/MCMF8KarO6xPg6J26W/giphy.gif)